### PR TITLE
Added info on start_rpc and container deployment for Cassandra [skip ci]

### LINKED ISSUE #976: Cassandra docs make no mention of setting start_rpc to true
It's come up a couple times in the google group and I think it makes sense to add a section on enabling start_rpc to the cassandra documentation. I'm also open to adding a section for deploying Cassandra in a docker container.

### PR DESCRIPTION
**Pull request by chupman**
fixes #976 

Signed-off-by: Chris Hupman <chupman@us.ibm.com>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there an issue associated with this PR? Is it referenced in the commit message?
- [X] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [X] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [X] Have you ensured that format looks appropriate for the output in which it is rendered?
- [X] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: docs/cassandra.adoc [hunk 1]
```diff
@@ -38,6 +39,13 @@ open();
 
 In the Gremlin Console, you can not define the type of the variables `conf` and `g`. Therefore, simply leave off the type declaration.
 
+=== Local Container Mode
+
+One of the easiest ways to run Cassandra on OSX or Windows is to use a Docker Container. You can download and run Cassandra with a single Docker command. In the command below an environment variable is being set to enable Cassandra Thrift with -e CASSANDRA_START_RPC=true.
```
Comment (pluradj) [inline docs/cassandra.adoc]: backticks around `-e CASSANDRA_START_RPC=true`

**Comment by chupman** [comment]
@pluradj requested changes have been made.

File: docs/cassandra.adoc [hunk 2]
```diff
@@ -38,6 +39,13 @@ open();
 
 In the Gremlin Console, you can not define the type of the variables `conf` and `g`. Therefore, simply leave off the type declaration.
 
+=== Local Container Mode
+
+Cassandra does not have a native install for Windows or OSX. One of the easiest ways to run Cassandra on OSX, Windows, or Linux is to use a Docker Container. You can download and run Cassandra with a single https://www.docker.com/[Docker] command. The https://hub.docker.com/_/cassandra/[Cassandra Docker Hub page] can be referenced for the available versions and useful commands. In the command below an environment variable is being set to enable Cassandra Thrift with `-e CASSANDRA_START_RPC=true`. A decription of the ports can be found https://docs.datastax.com/en/cassandra/latest/cassandra/configuration/secureFireWall.html[here]. Port 9160 is used for the Thrift client API. Port 9042 is for CQL native clients. Ports 7000, 7001 and 7099 are for inter-node communication.
```
Comment (analytically) [inline docs/cassandra.adoc]: specify which cassandra versions?

File: docs/cassandra.adoc [hunk 3]
```diff
@@ -26,6 +26,7 @@ image:modes-local.png[]
 Cassandra can be run as a standalone database on the same local host as JanusGraph and the end-user application. In this model, JanusGraph and Cassandra communicate with one another via a `localhost` socket. Running JanusGraph over Cassandra requires the following setup steps:
 
 . http://cassandra.apache.org/download/[Download Cassandra], unpack it, and set filesystem paths in `conf/cassandra.yaml` and `conf/log4j-server.properties`
+. If you want to be able to connect with the Gremlin Console you'll need to enable Cassandra Thrift. To do so open `conf/cassandra.yaml` and update `start_rpc: false` to `start_rpc: true`. If Cassandra is already running Thrift can be started manually with `bin/nodetool enablethrift`. the Thrift status can be verified with `bin/nodetool` statusthrift.
```
Comment (analytically) [inline docs/cassandra.adoc]: 'If you want to be able to connect with the' -> To?

File: docs/cassandra.adoc (hunk 2 above)
Comment (chupman) [reply docs/cassandra.adoc]: Added a link to the releases page and referenced where to find the supported versions of Cassandra.

File: docs/cassandra.adoc (hunk 3 above)
Comment (chupman) [reply docs/cassandra.adoc]: Reworded and pushed to [gh-pages on my fork](https://chupman.github.io/docs.janusgraph.org/latest/cassandra.html#_local_container_mode) to make review easier.

**Comment by pluradj** [comment]
@analytically Ready to give 2nd approval on this? Chris isn't a committer, and we need 2 approvals from committers.

**Pull request merged**